spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/RabbitConnectionFactoryBean.java

```diff
@@ -574,7 +575,9 @@ protected void setUpSSL() throws Exception {
 			}
 
 			if (this.logger.isDebugEnabled()) {
-				this.logger.debug("Initializing SSLContext with KM: " + keyManagers + ", TM: " + trustManagers
+				this.logger.debug("Initializing SSLContext with KM: "
+						+ (keyManagers == null? null : Arrays.asList(keyManagers))
```

Comment:
Whitespace before `?`.
OTOH why `Arrays.toString()` doesn't feet requirement? 
And it has built-in null check.

Just let me know and I'll fix on merge.